app-modules/inventory-management/src/Filament/Resources/AssetCheckInResource.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2022-2023, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\InventoryManagement\Filament\Resources;
+
+use Filament\Resources\Resource;
+use AdvisingApp\InventoryManagement\Models\AssetCheckIn;
+use AdvisingApp\InventoryManagement\Filament\Resources\AssetCheckInResource\Pages\EditAssetCheckIn;
+use AdvisingApp\InventoryManagement\Filament\Resources\AssetCheckInResource\Pages\ViewAssetCheckIn;
+use AdvisingApp\InventoryManagement\Filament\Resources\AssetCheckInResource\Pages\ListAssetCheckIns;
+use AdvisingApp\InventoryManagement\Filament\Resources\AssetCheckInResource\Pages\CreateAssetCheckIn;
+
+class AssetCheckInResource extends Resource
+{
+    protected static ?string $model = AssetCheckIn::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    protected static bool $shouldRegisterNavigation = false;
+
+    public static function getPages(): array
+    {
+        return [
+            'index' => ListAssetCheckIns::route('/'),
+            // 'create' => CreateAssetCheckIn::route('/create'),
+            // 'view' => ViewAssetCheckIn::route('/{record}'),
+            // 'edit' => EditAssetCheckIn::route('/{record}/edit'),
```

Comment:
Can these be removed or will they be needed soon?

Reply:
I would think these will be needed at some point in the near future. I can remove the references here but leave the stubbed files if that makes sense.

Reply:
If these are going to be needed soon then I am fine with it for right now. Thank you!